website/views/layouts/layout.ejs

```diff
@@ -306,6 +306,35 @@
     (see https://developer.mozilla.org/en-US/docs/Web/API/Window/self) */ %>
     <script>delete window.self;</script>
 
+    <script>
```

Comment:
Can you add a comment that describes what the purpose of this script tag is?

Reply:
Sure

Reply:
Done